pycode/memilio-generation/memilio/generation_test/test_data/test_oseir.cpp.txt

```diff
@@ -70,30 +77,60 @@ PYBIND11_MODULE(_simulation_test_oseir, m)
 
     pymio::bind_CustomIndexArray<mio::UncertainValue, mio::AgeGroup>(m, "AgeGroupArray");
 
-    pymio::bind_ParameterSet<mio::oseir::ParametersBase>(m, "ParametersBase");
+    pymio::bind_ParameterSet<mio::oseir::ParametersBase<double>, pymio::EnablePickling::Required>(m, "ParametersBase");
 
-    py::class_<mio::oseir::Parameters, mio::oseir::ParametersBase>(m, "Parameters")
-	.def(py::init<mio::AgeGroup>())
-	.def("check_constraints", &mio::oseir::Parameters::check_constraints)
-	.def("apply_constraints", &mio::oseir::Parameters::apply_constraints);
+    py::class_<mio::oseir::Parameters<double>, pymio::EnablePickling::Required, mio::oseir::ParametersBase<double>>(m, "Parameters");
```

Comment:
The following few lines dont seem to be right. There are too many semicolons at the end of lines. Compare it to before.